### PR TITLE
add spanish translations

### PR DESCRIPTION
**Pull request by JordiNeil**
## Description of the Change
Add spanish translations.
## Checklist

- [x] PR only contains one change (considered splitting up PR)
- [ ] unit-test added
- [ ] documentation updated
- [x] `CHANGELOG.md` updated (only for user relevant changes)
- [x] author name in `AUTHORS`

### CODE REVIEW

File: oauth2_provider/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+msgstr "Lista de URIs permitidas, separadas por espacio"
+
+#: oauth2_provider/models.py:113
+msgid "Hashed on Save. Copy it now if this is a new secret."
+msgstr "Encriptadas al guardar. Copiar ahora si este es un nuevo secreto."
+
+#: oauth2_provider/models.py:175
+#, python-brace-format
+msgid "Unauthorized redirect scheme: {scheme}"
+msgstr "Esquema de redirección no autorizado"
```
Comment (Andrew-Chen-Wang) [inline oauth2_provider/locale/es/LC_MESSAGES/django.po:71]: this should include the brackets

File: oauth2_provider/locale/es/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+#: oauth2_provider/models.py:179
+#, python-brace-format
+msgid "redirect_uris cannot be empty with grant_type {grant_type}"
+msgstr "redirected_uris no pueden estar vacías para el tipo de autorización {grant_type}"
```
Comment (Andrew-Chen-Wang) [inline oauth2_provider/locale/es/LC_MESSAGES/django.po:76]:
```suggestion
msgstr "redirect_uris no pueden estar vacías para el tipo de autorización {grant_type}"
```
variable name should stay the same

File: oauth2_provider/locale/es/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+msgstr "El token de acceso es válido pero no tiene suficiente alcance."
+
+#: oauth2_provider/templates/oauth2_provider/application_confirm_delete.html:6
+msgid "Are you sure to delete the application"
+msgstr "¿Está seguro de eliminar la aplicación"
+
+#: oauth2_provider/templates/oauth2_provider/application_confirm_delete.html:12
+#: oauth2_provider/templates/oauth2_provider/authorize.html:29
+msgid "Cancel"
+msgstr "Cancelar"
```
Comment (Andrew-Chen-Wang) [inline oauth2_provider/locale/es/LC_MESSAGES/django.po:105]: Seems like a command, so `Cancela`? i'm not hispanic, so just trying to recollect...

File: oauth2_provider/locale/es/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+#: oauth2_provider/templates/oauth2_provider/application_confirm_delete.html:13
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:38
+#: oauth2_provider/templates/oauth2_provider/authorized-token-delete.html:7
+msgid "Delete"
+msgstr "Eliminar"
```
Comment (Andrew-Chen-Wang) [inline oauth2_provider/locale/es/LC_MESSAGES/django.po:111]: `Elimina`? spanish command?

File: oauth2_provider/locale/es/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+msgstr "Tipo de acceso de autorización"
+
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:30
+msgid "Redirect Uris"
+msgstr "Uris de redirección"
+
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:36
+#: oauth2_provider/templates/oauth2_provider/application_form.html:35
+msgid "Go Back"
+msgstr "Volver"
```
Comment (Andrew-Chen-Wang) [inline oauth2_provider/locale/es/LC_MESSAGES/django.po:136]: command

File: oauth2_provider/locale/es/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:37
+msgid "Edit"
+msgstr "Editar"
```
Comment (Andrew-Chen-Wang) [inline oauth2_provider/locale/es/LC_MESSAGES/django.po:140]: command

File: oauth2_provider/locale/es/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+
+#: oauth2_provider/templates/oauth2_provider/application_form.html:9
+msgid "Edit application"
+msgstr "Editar aplicación"
+
+#: oauth2_provider/templates/oauth2_provider/application_form.html:37
+msgid "Save"
+msgstr "Guardar"
```
Comment (Andrew-Chen-Wang) [inline oauth2_provider/locale/es/LC_MESSAGES/django.po:148]: command

File: oauth2_provider/locale/es/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+msgid "Go Back"
+msgstr "Volver"
+
+#: oauth2_provider/templates/oauth2_provider/application_detail.html:37
+msgid "Edit"
+msgstr "Editar"
+
+#: oauth2_provider/templates/oauth2_provider/application_form.html:9
+msgid "Edit application"
+msgstr "Editar aplicación"
```
Comment (Andrew-Chen-Wang) [inline oauth2_provider/locale/es/LC_MESSAGES/django.po:144]: command

File: oauth2_provider/locale/es/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+msgid "Authorize"
+msgstr "Autorizar"
+
+#: oauth2_provider/templates/oauth2_provider/authorize.html:17
+msgid "Application requires the following permissions"
+msgstr "La aplicación requiere los siguientes permisos"
+
+#: oauth2_provider/templates/oauth2_provider/authorized-token-delete.html:6
+msgid "Are you sure you want to delete this token?"
+msgstr "¿Está seguro de que quiere eliminar este token?"
```
Comment (Andrew-Chen-Wang) [inline oauth2_provider/locale/es/LC_MESSAGES/django.po:185]: is there a difference between estas and esta? Translation of you is a bit all over the place, but I'm just nitpicking.

File: oauth2_provider/locale/es/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -0,0 +1,197 @@
...
+
+#: oauth2_provider/templates/oauth2_provider/authorized-tokens.html:6
+msgid "Tokens"
+msgstr "Tokens"
+
+#: oauth2_provider/templates/oauth2_provider/authorized-tokens.html:11
+msgid "revoke"
+msgstr "Anular"
```
Comment (Andrew-Chen-Wang) [inline oauth2_provider/locale/es/LC_MESSAGES/django.po:193]: command

**Review by Andrew-Chen-Wang — CHANGES_REQUESTED** [review]
Thanks!

sorry if I'm nitpicking too much; just remembering my days in spanish class

File: oauth2_provider/locale/es/LC_MESSAGES/django.po (hunk 3 above)
Comment (JordiNeil) [reply oauth2_provider/locale/es/LC_MESSAGES/django.po:105]: As you can see here, in spanish our verbs ends on "ar", "er", or "ir". The correct translation for "he cancel" is "él cancela" but for "to cancel" te translation is "Cancelar". In the next image you can see that windows tools use the verbs ending in ar, er, and ir. 

image

Comment (Andrew-Chen-Wang) [reply oauth2_provider/locale/es/LC_MESSAGES/django.po:105]: sounds good. Thanks!

**Review by Andrew-Chen-Wang — APPROVED** [review]
awesome. thanks for educating me 😅

File: oauth2_provider/locale/es/LC_MESSAGES/django.po (hunk 1 above)
Comment (JordiNeil) [reply oauth2_provider/locale/es/LC_MESSAGES/django.po]: Included in latest commit

**Pull request merged**